src/main/java/org/folio/marc4ld/util/MarcUtil.java

```diff
@@ -13,4 +16,11 @@ public static boolean isSubfieldPresent(char subfield, DataField dataField) {
   public static String getSubfieldValue(char subfield, DataField dataField) {
     return dataField.getSubfield(subfield).getData();
   }
+
+  public static void orderSubfields(DataField dataField) {
```

Comment:
Actually, this is an old method that have a bug. This logic will show numeric subfields before non-numeric sub fields. This logic is later replaced by `LetterFirstComparator`. We have to use LetterFirstComparator instead of this method.

I see that this method is still used in other places also. If you want, I an create a separate ticket to remove this method and fix it everywhere.

Reply:
fixed in this PR